pkg/metrics/listener.go

```diff
@@ -8,22 +8,31 @@ package metrics
 
 import (
 	"fmt"
-	"net"
+	"net/http"
+
+	"github.com/containerd/containerd/log"
+	"github.com/containerd/nydus-snapshotter/pkg/metrics/registry"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
 )
 
-// DefaultBindAddress sets the default bind address for the metrics
-// listener.
-var DefaultBindAddress = ":8080"
+// Endpoint for prometheus metrics
+var endpointPromMetrics = "/metrics"
```

Comment:
Should we change it to "/v1/metrics" like containerd ?
```
### Metrics API

The metrics API that outputs prometheus style metrics will be versioned independently,
prefixed with the API version. i.e. `/v1/metrics`, `/v2/metrics`.

The metrics API version will be incremented when breaking changes are made to the prometheus
output. New metrics can be added to the output in a backwards compatible manner without
bumping the API version.

```

Reply:
Yes, fixed it.